src/viam/sdk/robot/client.cpp

```diff
@@ -57,6 +53,146 @@ using viam::robot::v1::Status;
 // NOLINTNEXTLINE
 const std::string kStreamRemoved("Stream removed");
 
+// TODO: add a traits class for proto to type and back conversion
+DiscoveryQuery RobotClient::discovery_query::to_proto() const {
+    DiscoveryQuery proto;
+    *proto.mutable_subtype() = subtype;
+    *proto.mutable_model() = std::move(model);
```

Comment:
This `std::move` probably doesn't do anything: the method is `const` so `model` is const, so can't be moved from.